go/cmd/gh-ost/main.go

```diff
@@ -160,12 +160,15 @@ func main() {
 		flag.PrintDefaults()
 		return
 	}
+
```

Comment:
Let's always fallback to `unversioned`/`unknown` so that the `starting gh-ost %+v` also gets this fallback instead of `""`